reporting/miro_transformer/src/test_data/raw_data.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
I should have mentioned this on a previous review, but oh well: we probably want to be a little careful putting raw Miro records here, as some of them contain names/metadata we shouldn't be making public.